### PR TITLE
Added myTarget adapter files

### PR DESCRIPTION
**Pull request by aserrr**
Added myTarget adapter files.

### CODE REVIEW

**Comment by lostillusion** [comment]
I signed it!

**Comment by aserrr** [comment]
I signed it!

File: adapters/MyTarget/MyTargetAdapter/GADMAdapterMyTargetNative.m [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+	guard(_isContentAdRequested || _isAppInstallAdRequested) else
+	{
+		NSString *description = [NSString stringWithFormat:kGADMAdapterMyTargetErrorInvalidNativeAdType, _adTypesRequested];
+		MTRGLogError(description);
+		[strongConnector adapter:self didFailAd:[GADMAdapterMyTargetUtils errorWithDescription:description]];
+		return;
+	}
+
+	BOOL shouldDownloadImages = YES;
+	for (GADNativeAdImageAdLoaderOptions *imageOptions in options)
```
Comment (sbagadi) [inline adapters/MyTarget/MyTargetAdapter/GADMAdapterMyTargetNative.m]: The options are of type [`GADAdLoaderOptions`](https://developers.google.com/admob/ios/api/reference/Classes#/c:objc(cs)GADAdLoaderOptions). The sample code in the [Native Mediation Adapter](https://developers.google.com/admob/android/native-mediation) is incorrect. Please see the [SampleAdapter](https://github.com/googleads/googleads-mobile-ios-mediation/blob/8b819bf0063376920f0b7dc91487f2717894e8cd/example/Adapter/SampleAdapter.m#L155) for the correct implementation.

File: adapters/MyTarget/MyTargetAdapter/GADMAdapterMyTargetNative.m [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+}
+
+#pragma mark - MTRGNativeAdDelegate
+
+- (void)onLoadWithNativePromoBanner:(MTRGNativePromoBanner *)promoBanner nativeAd:(MTRGNativeAd *)nativeAd
+{
+	id<GADMAdNetworkConnector> strongConnector = _connector;
+	MTRGLogInfo();
+	guard(strongConnector) else return;
+
```
Comment (sbagadi) [inline adapters/MyTarget/MyTargetAdapter/GADMAdapterMyTargetNative.m]: Please add a check before creating a Mediated Native Ad to check if all the always included assets mentioned [here](https://support.google.com/admob/answer/6240809) are available. If any of the required assets are not available, the adapter should fail the request.

File: adapters/MyTarget/MyTargetAdapter/GADMAdapterMyTargetRewarded.m [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+	guard(strongConnector) else return;
+	[strongConnector adapterDidCloseRewardBasedVideoAd:self];
+}
+
+- (void)onVideoCompleteWithInterstitialAd:(MTRGInterstitialAd *)interstitialAd
+{
+	id<GADMRewardBasedVideoAdNetworkConnector> strongConnector = _connector;
+	MTRGLogInfo();
+	guard(strongConnector) else return;
+	NSNumber *amount = @0; //must not be nil
```
Comment (sbagadi) [inline adapters/MyTarget/MyTargetAdapter/GADMAdapterMyTargetRewarded.m]: The default reward amount needs to be 1. You could use `[NSDecimalNumber one]` for this.

File: adapters/MyTarget/MyTargetAdapter/GADMAdapterMyTargetNative.m (hunk 2 above)
Comment (aserrr) [reply adapters/MyTarget/MyTargetAdapter/GADMAdapterMyTargetNative.m]: Requested changes has been made.

**Pull request merged**